cmd/clusterctl/client/repository/metadata_client.go

```diff
@@ -92,7 +92,39 @@ func (f *metadataClient) Get(ctx context.Context) (*clusterctlv1.Metadata, error
 		return nil, errors.Wrapf(err, "error decoding %q for provider %q", metadataFile, f.provider.ManifestLabel())
 	}
 
-	//TODO: consider if to add metadata validation (TBD)
+	if err != nil {
```

Comment:
This if seems like unreachable code. The error is already checked above

Reply:
Thx. I fixed it.